tools/dds/dds-server/CMakeLists.txt

```diff
@@ -3,17 +3,21 @@
 cmake_minimum_required( VERSION 3.1.0 )
 project( rs-dds-server )
 
+include(../../../common/CMakeLists.txt)
```

Comment:
Why? All it does it set some variables...

Reply:
Removed. Leftovers of when I hoped I could use it instead of naming individual files (metadata-helper.cpp)